### ISSUE TITLE
Mouse Preferences dialog too large to fit on netbook screens

### ISSUE THREAD

**Issue by bplu4t2f** [mate-desktop/mate-control-center#350]
#### Expected behaviour

Mouse Preferences dialog should fit on a 1024x600 netbook screen (taking panel height into consideration), or at least be resizable with scoll bars.

#### Actual behaviour

Window too high, can't be resized; Help/Close buttons at the bottom just barely not visible.

#### MATE general version

1.18

#### Package version

1.18.2-1+sonya I guess?

#### Linux Distribution

mint

**Comment by tamplan** [comment]
Tested with 1024x768 results:
![Capture du 2019-06-13 00-20-42](https://user-images.githubusercontent.com/50292710/59390321-34502800-8d71-11e9-887e-4ec7d8904d95.png)
Is the bug persists for you?

**Comment by bplu4t2f** [comment]
I tested it on an Asus eee PC with 1024x600, not 1024x768. Maybe newer netbook models have higher screen resolution, but for the older generations, 600 high was very common.

Many programs were not usable on the 600 high screen, so I eventually gave up.

**Comment by lukefromdc** [comment]
All of mine were 600X1024, workaround was to run compiz and expo so I could push them up and partially out of the visible workspace and expose the bottom area. Be sure to set a touchpad setting you can actually use to drag a window while in expo for this

**Comment by flexiondotorg** [comment]
I don't think we should be targetting the resolution of 1024x600 to cater to 10+ year old netbooks.

The two most popular resolutions for Ubuntu users today are 1366x768 and 1920x1080 with less than 0.5% of users having a vertical resolution of 600 pixels.

The current generation of "netbook" form factor devices (from the like of GPD and TopJoy) are shipping with 1920x1080 or 1920x1200.

I think targeting a minimum vertical resolution of 768 is reasonable.

**Comment by Mas4hmad** [comment]
![VirtualBox_other_12_02_2023_01_29_03](https://user-images.githubusercontent.com/45534607/218275456-78932a72-2662-40e5-a47d-0f2199101bea.png)
In openindiana (hipster 2022-11). Seem like the window size has increased. So, it won't fit in 1024x768

**Comment by lukefromdc** [comment]
Last commit adding an item vertically to that window is 96e46794a82bf4b2a3edce4502f09f1c4ec85678 from August 2, 2019. This was in release 1.24.0 released on Feb 8 2020. If you ever had this window in a smaller dimension than this with mate-panel 1.24.0 or later, the cause didn't come from a code change in the mate-settings-daemon mouse preferences capplet
